packages/js/product-editor/src/blocks/generic/text/block.json

```diff
@@ -27,21 +24,29 @@
 		"tooltip": {
 			"type": "string"
 		},
-		"required": {
-			"type": "boolean",
-			"default": false
+		"suffix": {
+			"type": "object"
 		},
-		"validationRegex": {
-			"type": "string"
+		"type": {
+			"type": "object"
 		},
-		"validationErrorMessage": {
-			"type": "string"
+		"required": {
+			"type": "object"
```

Comment:
I think this may also support an array, as this should be this right? `"type": [ "string", "boolean" ],`?

Reply:
Oh really!!? don't like it but good to have it.

Done -> https://github.com/woocommerce/woocommerce/pull/41442/commits/fe5f81732ab419cab5e6c67200ccd913424ac717

Reply:
I had to revert that due to conflicts with TypeScript